tox.ini

```diff
@@ -12,25 +12,26 @@ envdir =
     pypy3: {toxworkdir}/pypy3
     py: {toxworkdir}/py
 commands =
-    python housekeep.py setup
-    nocov: python -m nose2 -v {posargs}
-    cov: python -m coverage run -m nose2
-    cov: python -m coverage combine
-    cov: python -m coverage html --directory=htmlcov/{envname}
-    cov: python -m coverage report -m --fail-under=100
-    cov: python -m coverage xml
-    diffcov: diff-cover coverage.xml --html-report diffcov.html
-    diffcov: diff-cover coverage.xml --fail-under=100
-    profile: python -m nose2 --profile {posargs}
+    python housekeep.py prep
+    nocov: pytest --verbose -p no:cov {posargs}
+    cov: pytest --cov --cov-report=xml --cov-report=html --cov-report=term {posargs}
+    diffcov: diff-cover coverage-{env:INTERP}.xml --html-report diffcov-{env:INTERP}.html
+    diffcov: diff-cover coverage-{env:INTERP}.xml --fail-under=100
+    profile: pytest --profile {posargs}
     python housekeep.py gather
 #sitepackages = True
 usedevelop = True
 deps =
-    nose2
-    coverage
+    packaging
+    pytest
+    pytest-cov
+    pytest-mock
+    pytest-profiling
+    pytest-sugar
     diff_cover
 setenv =
-    cov: COVERAGE_PROCESS_START=.coveragerc
+    cov: COVERAGE_RCFILE=coverage.cfg
+    cov: COVERAGE_PROCESS_START=coverage.cfg
```

Comment:
`.coveragerc` has been renamed to `coverage.cfg`

See 454ce0b4

Reply:
... and this is now irrelevant because we've folded coverage.py's config into `pyproject.toml`